### PR TITLE
BZ2105924: Memory Manager support status updated from TP to GA in 4.9

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s):
4.9

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2105924
(Clone of https://bugzilla.redhat.com/show_bug.cgi?id=2105152)

Link to docs preview:
http://file.emea.redhat.com/amolnar/BZ2105924-4.9/release_notes/ocp-4-9-release-notes.html#ocp-4-9-scale-memory-manager-in-beta-version
http://file.emea.redhat.com/amolnar/BZ2105924-4.9/release_notes/ocp-4-9-release-notes.html#ocp-4-9-technology-preview

Additional information:

### CODE REVIEW

**Comment by mrniranjan** [comment]
LGTM

**Review by mjpytlak — COMMENTED** [review]
Peer review LGTM.

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -1944,10 +1944,10 @@ In the table below, features are marked with the following statuses:
 |TP
 |GA
 
-|Memory Manager feature
+|Memory Manager
 |-
 |-
-|TP
+|GA
```
Comment (fromanirh) [inline release_notes/ocp-4-9-release-notes.adoc:1950]: I don't know the proper way so let me ask: shouldn't it be TP in the previous version?

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -845,8 +845,8 @@ You can now use the Special Resource Operator (SRO) to help manage the deploymen
 For more information, see xref:../hardware_enablement/psap-special-resource-operator.adoc#about-special-resource-operator_special-resource-operator[About the Special Resource Operator].
 
 [id="ocp-4-9-scale-memory-manager-in-beta-version"]
-==== Memory Manager feature (Technology Preview)
-The Memory Manager feature is now enabled by default for all pods running on the node that is configured with one of the following Topology Manager policies:
+==== Memory Manager is generally available
+The Memory Manager, a subcomponent of the Performance Addon Operator, is now enabled by default for all pods running on the node that is configured with one of the following Topology Manager policies:
```
Comment (fromanirh) [inline release_notes/ocp-4-9-release-notes.adoc:849]: the memory manager is a component of the kubelet (https://kubernetes.io/docs/tasks/administer-cluster/memory-manager/) configured using the Performance Addon Operator (up to OCP 4.10) or with the Node Tuning Operator (OCP 4.11 and onwards)

**Review by ffromani — CHANGES_REQUESTED** [review]
there's something to fix: the Memory Manager is part of the kubelet, not PAO

File: release_notes/ocp-4-9-release-notes.adoc (hunk 2 above)
Comment (swatisehgal) [inline release_notes/ocp-4-9-release-notes.adoc:849]: I don't think it is appropriate to say that Memory Manager is a subcomponent of the Performance Addon Operator. Memory manager is a Kubelet component and Performance Addon Operator is a way of configuring `kubeletconfig` with certain memory manager settings.

In addition to that, [Performance Addon Operator](https://github.com/openshift-kni/performance-addon-operators) has been deprecated and is moved to [Node Tuning Operator](https://github.com/openshift/cluster-node-tuning-operator) under that name of  referred to as [Performance Profile Controller](https://github.com/openshift/cluster-node-tuning-operator#perfomance-profile-controller).

Comment (MarSik) [reply release_notes/ocp-4-9-release-notes.adoc:849]: (just as a side note: NTO is what should be in the docs, performance profile controller or operator terminology is now internal only)

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (amolnar-rh) [reply release_notes/ocp-4-9-release-notes.adoc:1950]: @fromanirh Based on the bug's description, my understanding was that this feature was supposed to be in GA status in 4.9 as well. Link to original bug: https://bugzilla.redhat.com/show_bug.cgi?id=2105152
@fbaudin, you've lgtm'd this change, could you please clarify?

File: release_notes/ocp-4-9-release-notes.adoc (hunk 2 above)
Comment (amolnar-rh) [reply release_notes/ocp-4-9-release-notes.adoc]: @fromanirh @swatisehgal Updated. WDYT?
> The Memory Manager, a kubelet subcomponent configured by the Performance Addon Operator, is now enabled by default for all pods running on the node that is configured with one of the following Topology Manager policies:

File: release_notes/ocp-4-9-release-notes.adoc [hunk 3]
```diff
@@ -846,7 +846,7 @@ For more information, see xref:../hardware_enablement/psap-special-resource-oper
 
 [id="ocp-4-9-scale-memory-manager-in-beta-version"]
 ==== Memory Manager is generally available
-The Memory Manager, a subcomponent of the Performance Addon Operator, is now enabled by default for all pods running on the node that is configured with one of the following Topology Manager policies:
+The Memory Manager, a kubelet subcomponent configured by the Performance Addon Operator, is now enabled by default for all pods running on the node that is configured with one of the following Topology Manager policies:
```
Comment (fromanirh) [inline release_notes/ocp-4-9-release-notes.adoc:849]: this looks good to me, thanks!

File: release_notes/ocp-4-9-release-notes.adoc (hunk 2 above)
Comment (amolnar-rh) [reply release_notes/ocp-4-9-release-notes.adoc]: Only PTO is mentioned in the 4.9, and that should be okay. The 4.10 version of this bug only affects the tech preview table that doesn't mention PTO or NTO or performance profile controller

**Comment by fromanirh** [comment]
LGTM

**Pull request merged**